task-launcher/src/tasks/shared/helpers/initTrialSaving.js

```diff
@@ -37,7 +38,11 @@ export const initTrialSaving = (config) => {
       dataCopy.responseSource = data.response_source;
       delete dataCopy.trial_index;
       delete dataCopy.trial_type;
-      config.firekit.writeTrial(dataCopy);
+      if (config.task === 'trog') {
+        config.firekit.writeTrial(dataCopy, computedScoreCallback);
+      } else {
+        config.firekit.writeTrial(dataCopy);
+      }
```

Comment:
Can you make this conditional only for ROAR. We won't need it. I think it would be good to have a ROAR param for cases like this as well as any future ROAR changes / versions of core tasks.